packages/react-components/src/components/VideoTile.tsx

```diff
@@ -404,6 +406,12 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           /* @conditional-compile-remove(reaction) */
           reactionOverlay
         }
+        {
+          /* @conditional-compile-remove(ppt-live) */
+          //TODO remove when pptlive local navigation feature is ready
```

Comment:
Update comment to say this can be removed once the overlay mitigation has been implemented at the SDK layer